packages/embed/src/types.ts

```diff
@@ -27,6 +27,7 @@ export type Config = {
   onCustomSubmit?: ({ method }) => void
   metadata?: Record<string, string>
   paymentSource?: 'installment' | 'recurring'
+  cartItems?: Array<Record<string, unknown>>
```

Comment:
I think the interface is important for library consumers - https://github.com/gr4vy/embed-ui/pull/142/files
Embed UI doesn't need to know - but end users using TS will benefit from it.